src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -19,11 +19,6 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Release|AnyCPU'" />
```

Comment:
NIT: you still want to keep the uapaot one. I suggest you just run `msbuild /t:UpdateVSConfigurations build.proj` from the root so that you know that all of them are refreshed correctly.